tasks/installer.py

```diff
@@ -79,13 +79,22 @@ def build(
 @task
 def build_linux_script(
     ctx,
+    output_bin=None,
```

Comment:
Is there a use case for not always generating the script in the expected (from the packaging POV) location?

Reply:
You mean always generating in `bin/installer/...` and then doing a copy from gitlab or always generating in `$OMNIBUS/<package>` ?

Reply:
I guess always generating in `OMNIBUS_PACKAGE_DIR` since that's what the CI would do after this PR

But more generally, is there a use case for supporting multiple output directories?